source/common/stats/utility.h

```diff
@@ -34,6 +58,160 @@ class Utility {
    * @return The value of the tag, if found.
    */
   static absl::optional<StatName> findTag(const Metric& metric, StatName find_tag_name);
+
+  /**
+   * Creates a counter from a vector of tokens which are used to create the
+   * name. The tokens can be specified as string_view or StatName. For
```

Comment:
should `DynamicName` be mentioned here instead of implying that string_views can be passed directly? Or is that intentional?

Reply:
Yes, that was a doc mistake. Cleaned up. Good catch.